rovilay/SMS-management
# Ft contact

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?
* setups contact model
* creates contact
*  logins contact
*  deletes contact

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/SMS-management.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ch-setup-server`
- set up the `.env` with the `.env.sample` file
- install dependencies - `yarn install`
- start the app - `yarn run server`
- navigate to `localhost:port/api/v1/contacts/signup` on postman to signup
- you should have an account

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
